nix/process-compose.nix

```
@@ -0,0 +1,11 @@
+{ inputs, ... }:
+{
+  perSystem = {
+    process-compose.services = {
```

To avoid the "services.services" option namespace confusion:

```suggestion
    process-compose.todo-app-stack = {
```